server/src/repositories/person.repository.ts

```diff
@@ -457,6 +457,16 @@ export class PersonRepository {
     return result?.latestDate;
   }
 
+  @GenerateSql({ params: [DummyValue.UUID] })
```

Comment:
This won't work. The function expects a face object and not a single uuid string